workflow_calcium_imaging/__init__.py

```diff
@@ -0,0 +1,26 @@
+try:
+    import matlab.engine
+except Exception:
+    pass
+
+import os
+import datajoint as dj
+
+if "custom" not in dj.config:
+    dj.config["custom"] = {}
+
+# overwrite dj.config['custom'] values with environment variables if available
```

Comment:
What's the motivation for these adds?

Reply:
All the changes are transferred from `wt-calcium-imaging` as it is the most cloud-tested calcium imaging workflow.

Reply:
The priority is given to environment variables. If the variables are not available, these adds set the defaults.